package.json

```diff
@@ -14,45 +14,45 @@
     "generate-api-schema": "node utils/generateAPISchema.js"
   },
   "dependencies": {
-    "@google-cloud/firestore": "^4.9.1",
+    "@google-cloud/firestore": "^4.9.4",
     "axios": "^0.21.1",
-    "config": "^3.3.2",
-    "cookie-parser": "~1.4.4",
+    "config": "^3.3.3",
+    "cookie-parser": "~1.4.5",
     "cors": "^2.8.5",
-    "debug": "~2.6.9",
-    "express": "~4.16.1",
+    "debug": "~4.3.1",
+    "express": "~4.17.1",
     "express-boom": "^3.0.0",
-    "helmet": "^4.0.0",
-    "http-errors": "~1.6.3",
-    "joi": "^17.2.1",
+    "helmet": "^4.4.1",
+    "http-errors": "~1.8.0",
+    "joi": "^17.4.0",
     "jsdoc": "^3.6.6",
     "jsonwebtoken": "^8.5.1",
-    "morgan": "~1.9.1",
+    "morgan": "~1.10.0",
     "passport": "^0.4.1",
     "passport-github2": "^0.1.12",
-    "swagger-jsdoc": "^4.0.0",
-    "swagger-ui-express": "^4.1.4",
+    "swagger-jsdoc": "^6.0.6",
+    "swagger-ui-express": "^4.1.6",
     "winston": "^3.3.3"
   },
   "devDependencies": {
-    "chai": "^4.2.0",
+    "chai": "^4.3.0",
     "chai-http": "^4.3.0",
-    "cross-env": "^7.0.2",
-    "eslint": "^7.9.0",
-    "eslint-config-standard": "^14.1.1",
-    "eslint-plugin-import": "^2.22.0",
+    "cross-env": "^7.0.3",
+    "eslint": "^7.20.0",
+    "eslint-config-standard": "^16.0.2",
+    "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-mocha": "^8.0.0",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-promise": "^4.3.1",
     "eslint-plugin-security": "^1.4.0",
-    "eslint-plugin-standard": "^4.0.1",
-    "mocha": "^8.1.1",
-    "nock": "^13.0.5",
-    "nodemon": "^2.0.4",
+    "eslint-plugin-standard": "^4.1.0",
+    "mocha": "^8.3.0",
+    "nock": "^13.0.7",
+    "nodemon": "^2.0.7",
     "nyc": "^15.1.0",
     "pre-commit": "^1.2.2",
-    "sinon": "^9.0.3",
-    "standard": "^14.3.4"
+    "sinon": "^9.2.4",
+    "standard": "^16.0.3"
```

Comment:
Are we using this package now?

Reply:
I read up the docs for this package and it is used an alternative to using ESLint. 
Since we're already using ESLint, this can be removed. I tried to run `lint` script and it works just fine. Getting output same as mentioned in the screenshot above

Reply:
Removed it.